packages/autocomplete-plugin-algolia-insights/src/createAlgoliaInsightsPlugin.ts

```diff
@@ -23,7 +23,7 @@ type SendViewedObjectIDsParams = {
   onItemsChange(params: OnItemsChangeParams): void;
   items: AlgoliaInsightsHit[];
   insights: InsightsApi;
-  state: AutocompleteState<unknown>;
+  state: AutocompleteState<any>;
```

Comment:
could you clarify why the state moved to any?

Reply:
It should have been in the merge commit.

The core now requires that `TItem` extends `BaseItem` (which is a `Record`). `unknown` is not compatible with `TItem`. Since we're not doing anything with `TItem` here, I think `any` is legit.